conanfile.py

```diff
@@ -32,6 +32,9 @@ def requirements(self):
         if self.options.with_bzip2:
             self.requires.add("bzip2/1.0.6@conan/stable")
 
+        if self.settings.os == "Windows" or self.settings.os == "Linux":
```

Comment:
OpenSSL could be add by an option:

```
```

Reply:
Agreed, Default to True or False ?

Reply:
True, because libzip uses True by default:

Reply:
if option is only with_ssl, then it should check os because recommanded library for osx is not openssl but CommonCrypto